internal/monitors/vsphere/service/log.go

```diff
@@ -0,0 +1,5 @@
+package service
+
+import "github.com/sirupsen/logrus"
+
+var Log = logrus.WithField("monitorType", "vsphereInfo")
```

Comment:
Does Log need to be exported?

Reply:
Yeah because it's used by the `vsphere` package. Not sure if this is an idiomatic way to log though.

Reply:
A better pattern that existing agent code isn't terribly consistent with is to have the logger be instantiated in the `Configure` method with fields for the monitor id (`conf.MonitorID`) and type being equal to the registered monitor type (`vsphere`, not `vsphereInfo`).  Then pass that logger instance around to the various structs that need logging.  This allows you to distinguish between multiple instances of the monitor is the case ever arose.

In general it is best to try and "push up" errors to the top level of the monitor or current go routine and log them there but sometimes for things like info/debug messages you will need the logger instance in child structs of the monitor so you will have to put it on those structs.

Reply:
Sounds good, will do this.